apps/web/app/(app)/gardens/[chain]/[garden]/pool/[poolId]/proposals/[proposalId]/page.tsx

```diff
@@ -239,11 +282,11 @@ export default async function Proposal({
           </div>
           <div className=" flex items-baseline justify-end space-x-4 ">
             <h3 className="w-full text-center text-2xl font-semibold">
-              {title}
+              {ipfsResult?.title}
```

Comment:
spinner?

Reply:
The nextjs loader is for pages not async fetch but i can make it loads while fetching ipfs yeah

Reply:
https://github.com/1Hive/gardens-v2/pull/229/commits/7de7a961e5d2a1ad87b6119713f6e277cab5ef5c